src/lib/features/feature-toggle/feature-toggle-controller.ts

```diff
@@ -1060,6 +1062,9 @@ export default class ProjectFeaturesController extends Controller {
         const strategy = await this.featureService.getStrategy(strategyId);
 
         const { newDocument } = applyPatch(strategy, patch);
+
+        throwOnInvalidSchema(newDocument, featureStrategySchema.$id);
```

Comment:
we can call it validateSchema since it's a pattern we follow in other places. Internally those functions return void and throw and error but we describe business purpose (validate) not implementation details (throw) in the method signature

Reply:
We already have a `validateSchema` in this file, but I'm open to suggestions!